Content.Server/SS220/Storage/SpawnOnOpen/Components/SpawnItemsOnOpenComponent.cs

```diff
@@ -0,0 +1,18 @@
+using Robust.Shared.Prototypes;
+using Content.Shared.EntityTable.EntitySelectors;
+
+namespace Content.Server.SS220.Storage.SpawnOnStorageOpen.Components;
+
+[RegisterComponent]
+public sealed partial class SpawnOnStorageOpenComponent : Component
+{
+    /// <summary>
+    ///     Basically any class inheriting from <see cref="EntityTableSelector"/> will fit
+    /// </summary>
+    [DataField("selector")]
+    public EntityTableSelector Selector;
```

Comment:
Если совпадает с названием переменной - то можно не указывать
`required: true` - для того чтобы при добавлении компонента в прототип обязательно указывали значение этого поля
```suggestion
    [DataField(required: true)]
    public EntityTableSelector Selector;
```

Reply:
Компилятор кста не выдаёт ошибок из-за того, что полю не присвоено значение (на момент инициализации класса)?